frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-runs.feature

```diff
@@ -336,22 +337,40 @@ Feature: Pipeline Runs
               And user navigates to pipelineRun logs tab
              Then user is able to see log snippet for failure of "build-image" task
 
-        @regression
+        @regression @odc-4793
```

Comment:
Already added the tag `@odc-4793` here in this PR https://github.com/openshift/console/pull/11920/files#diff-8e210ca906190602a60e37911cb29f65c9b98d6ed41ba3543b1cf442d29d042eR339. As we discussed before.

Reply:
okay